Doc/library/statistics.rst

```diff
@@ -35,6 +35,34 @@ and implementation-dependent.  If your input data consists of mixed types,
 you may be able to use :func:`map` to ensure a consistent result, for
 example: ``map(float, input_data)``.
 
+Some datasets use ``NaN`` (not a number) values to represent missing data.
+Since NaNs have unusual comparison semantics, they cause surprising or
+undefined behaviors in the statistics functions that sort data or that count
+occurrences.  The functions affected are median, median_low, median_high,
+median_grouped, mode, multimode, and quantiles.  The NaN values should be
```

Comment:
Missing markup:

```suggestion
occurrences.  The functions affected are :func:`median`, :func:`median_low`,                     
:func:`median_high`, :func:`median_grouped`, :func:`mode`, :func:`multimode`,                                                     
and :func:`quantiles`.  The ``NaN`` values should be
```

Reply:
I don't think these need links.  We have those in an organized fashion immediately below.

Reply:
Fair enough. Something like this would be an improvement, in my opinion, though:

```suggestion
occurrences.  The functions affected are ``median()``, ``median_low()``,                     
``median_high()``, ``median_grouped()``, ``mode()``, ``multimode()``,                                                     
and ``quantiles()``.  The ``NaN`` values should be
```